NEWS

```diff
@@ -7,6 +7,15 @@
 ##
 #
 
+## 1.8.1-RC1 (June 23, 2020)
```

Comment:
also need to bum version in confugure.ac to 1.8.1

Reply:
Good point. Any other places that I have missed

Reply:
no, this is only place, doc are generated automatically